i18n/en.json

```diff
@@ -29,12 +29,12 @@
 	"logentry-farmer-requestwiki": "$1 requested the wiki \"$4\" (language: $5; private: $6) in $7",
 	"log-name-farmer": "Farmer log",
 	"requestwiki": "Request a wiki",
-	"requestwiki-done": "Your request has successfully been submitted.",
+	"requestwiki-done": "Your request has successfully been submitted. Please note that it will be verified by a human to prevent spam, this shouldn't take more than 48 hours.",
 	"requestwiki-error-csrf": "<strong>Sorry, the request could not be submitted due to a loss of session data.</strong>\nPlease submit the form again. If that doesn't work, please [[Special:Userlogout|log out]] and log back in.", 
 	"requestwiki-error-nodomain": "You must specify either a subdomain or custom domain.",
 	"requestwiki-error-notallfilledin": "You have to fill in at least the wiki name and language fields.",
 	"requestwiki-error-twodomains": "You specified both a subdomain and custom domain: you can only choose between one of them.",
-	"requestwiki-header": "You are going to request a new wiki at Miraheze, great! We need you to fill in this form, before we can review your request.\n\nIf you want a custom domain for your wiki (i.e. not a standard .miraheze.org subdomain), please read <span class=\"plainlinks\">[$1 this]</span> first.",
+	"requestwiki-header": "You are going to request a new wiki at Miraheze, great! We need you to fill in this form, before we can review your request./n Please note that this your request will be reviewed by a human for preventing spam.\n\nIf you want a custom domain for your wiki (i.e. not a standard .miraheze.org subdomain), please read <span class=\"plainlinks\">[$1 this]</span> first. ",
```

Comment:
Per NDK, and please rewrite the sentence to:
"Your wiki request will be reviewed by a volunteer, and it can be tracked at the link given when submitting your request."